### PR TITLE
docs(support-matrix): update support matrix

### PR DESCRIPTION
**Pull request by Ankurk99**
**Purpose of PR**:
Update support matrix to reflect recently added Distributions

**Does this PR introduce a breaking change?**
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
Yes, all the updated distributions were manually tested

**Checklist:**
- [x] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`

### CODE REVIEW

File: getting-started/support_matrix.md [hunk 1]
```diff
@@ -19,6 +19,23 @@ KubeArmor supports following types of workloads:
...
+## Supported Linux Distributions
+
+| Provider | Distro | VM / Bare-metal | Kubernetes |
+|----------|--------|---------------|------|
+| SUSE | SUSE Enterprise 15 | Full | Full |
+| Debian | [Buster](https://www.debian.org/releases/buster/) / [Bullseye](https://www.debian.org/releases/bullseye/) | Full | Full |
+| Ubuntu | 18.04 / 20.04 | Full | Full |
+| RedHat / CentOS | RHEL 8.4 / CentOS 8.4 | Full | Partial |
+| RedHat | RHEL 9 | Full | Full |
+| CentOS | CentOS 8.5 | Full | Full |
```
Comment (daemon1024) [inline getting-started/support_matrix.md]:
```suggestion
| RedHat | RHEL 9 / RHEL ^8.5 /  Centos 8 Stream | Full | Full |
```

File: getting-started/support_matrix.md [hunk 2]
```diff
@@ -19,6 +19,23 @@ KubeArmor supports following types of workloads:
...
+| Fedora | Fedora 34 / 35 | Full | Full |
+| Rocky Linux | Rocky Linux 8.7 | Full | Full |
```
Comment (daemon1024) [inline getting-started/support_matrix.md]:
```suggestion
| Rocky Linux | Rocky Linux ^8.5 | Full | Full |
```

Comment (Ankurk99) [reply getting-started/support_matrix.md]: What I mentioned here are the distros on which we have tried KubeArmor. If we want to mention the theoretically supported distros then we can add a lot more. Should we do that?

File: getting-started/support_matrix.md [hunk 3]
```diff
@@ -19,6 +19,23 @@ KubeArmor supports following types of workloads:
...
 | Nutanix | * | TBD |
 
+## Supported Linux Distributions
+
+| Provider | Distro | VM / Bare-metal | Kubernetes |
+|----------|--------|---------------|------|
+| SUSE | SUSE Enterprise 15 | Full | Full |
+| Debian | [Buster](https://www.debian.org/releases/buster/) / [Bullseye](https://www.debian.org/releases/bullseye/) | Full | Full |
+| Ubuntu | 18.04 / 20.04 | Full | Full |
+| RedHat / CentOS | RHEL 8.4 / CentOS 8.4 | Full | Partial |
```
Comment (nyrahul) [inline getting-started/support_matrix.md:29]: I am not sure if we should mention Full support at all in case of SELinux only systems.

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: getting-started/support_matrix.md (hunk 3 above)
Comment (Ankurk99) [reply getting-started/support_matrix.md:29]: I have tried it on RHEL 8.5 with BPF LSM disabled and enforcement is running fine with SELinux on Host based policy.

```
== Alert / 2022-11-29 18:39:07.163303 ==
HostName: ankur-test
Type: MatchedHostPolicy
PolicyName: hsp-kubearmor-dev-file-path-block
Severity: 5
Source: /usr/bin/cat /etc/hostname
Resource: /etc/hostname
Operation: File
Action: Block
Data: syscall=SYS_OPENAT fd=-100 flags=O_RDONLY
Enforcer: SELinux
Result: Permission denied
HostPID: 3627
HostPPID: 2895
PID: 3627
PPID: 2895
ParentProcessName: /usr/bin/bash
ProcessName: /usr/bin/cat
UID: 1000
```

Comment (Ankurk99) [reply getting-started/support_matrix.md:29]: image

**Review by daemon1024 — APPROVED** [review]

**Review by nyrahul — APPROVED** [review]

**Pull request merged**